src/signals/incident-management/containers/FilterTagList/__tests__/FilterTagList.test.js

```diff
@@ -126,7 +110,28 @@ describe('signals/incident-management/containers/FilterTagList', () => {
         ),
       );
 
-      expect(container.querySelectorAll('span')).toHaveLength(6);
+      expect(queryByText('Normaal')).toBeTruthy();
```

Comment:
Using `.toBeInTheDocument()` has the same result, but is more descriptive. I would up opt for using that.

Reply:
I agree